.gitignore

```diff
@@ -33,4 +33,4 @@ out/
 .vscode/
 
 db/data/
-production.env
\ No newline at end of file
+skeleton.env
```

Comment:
This was intended. skeleton.env is a sample configuration, which should reside in the project. The `production.env`, on the other hand, should never be tracked by Git. So please revert these changes.

Reply:
And re-add the skeleton.env file please

Reply:
I'm sorry i guess this happend when i used rename function